src/model_DAforcing.py

```diff
@@ -369,10 +372,22 @@ def run(self, values: dict):
                 self._t0,
                 lastobs_output_folder
             )
+  
+        # if output_parameters.get('stream_output',{}) is not None and output_parameters.get('stream_output',{}).get('qvd_nudge'):
+        if output_parameters.get('stream_output',{}):
+            
+            t0 = self._t0
+            stream_output_directory =  output_parameters.get('stream_output').get('stream_output_directory')
+            stream_output_type = output_parameters.get('stream_output').get('stream_output_type')
+            stream_output_internal_frequency = output_parameters.get('stream_output').get('stream_output_internal_frequency')
```

Comment:
You might need to reformulate these checks. Based on the default values for these parameters, I think a user could cause an error if they have something like
```
stream_output:
    #stream_output_directory:
```
In this case `stream_output_directory` would be `None` which I believe would cause a problem in the write function.

Reply:
That's true. But should we catch it with pydantic before it starts though?